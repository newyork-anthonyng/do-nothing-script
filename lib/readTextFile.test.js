jest.mock('yargs', () => {
  const self = {};
  self.alias = () => self;
  self.demandOption = () => self;
  self.describe = () => self;
  self.argv = {
    file: './lib/__mocks__/someFile.js'
  };

  return self;
});

const readTextFile = require('./readTextFile');

it('should return file correct file', () => {
  const result = readTextFile();

  expect(result.length).toEqual(4);
  expect(result).toMatchSnapshot();
});