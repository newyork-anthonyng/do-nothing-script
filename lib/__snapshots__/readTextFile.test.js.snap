// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return file correct file 1`] = `
Array [
  "1: git init",
  "2: git commit",
  "3: git push",
  "4: git paid",
]
`;
